src/visualization/Turi Create Visualization/src/js/lib/index.js

```diff
@@ -146,6 +146,36 @@ window.updateData = function updateData(data) {
     }
 }
 
+window.inputJSONHandler = function(data){
```

Comment:
Let's name this function with a verb instead of a noun: `handleInput` (the "JSON" part is redundant/incorrect, as all JavaScript data structures are serializable to/from JSON, and this particular one comes in as a deserialized object).